SpritesOut/SpritesOut.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace SpriteOut
+{
+    public class SpriteCSharpOut
+    {
+        private static void Main()
```

Comment:
The problem with the main is that the project always has only 1 entry point into the main.
And other objects should be launched from it.
Therefore, you need to simply create classes with methods in different projects. And in the main of one of the projects call these classes